sklearn/naive_bayes.py

```diff
@@ -838,8 +861,11 @@ class MultinomialNB(_BaseDiscreteNB):
     [3]
```

Comment:
Modifying docstrings' examples to add `force_alpha=True`, e.g. here:

```
    >>> import numpy as np
    >>> rng = np.random.RandomState(1)
    >>> X = rng.randint(5, size=(6, 100))
    >>> y = np.array([1, 2, 3, 4, 5, 6])
    >>> from sklearn.naive_bayes import MultinomialNB
    >>> clf = MultinomialNB()
    >>> clf.fit(X, y)
    MultinomialNB(force_alpha=True)
    >>> print(clf.predict(X[2:3]))
    [3]
```

should resolve the last issues on the CI.

Reply:
It's fine, as non-Windows users, we are easily able to catch errors on Windows as well. So it's good to have someone in the team which uses this OS. 🙂